tutorials/object_spark_1_intro.ipynb

```diff
@@ -0,0 +1,682 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For now, we have `extendedness == base_ClassificationExtendedness_value` as a tool for star/galaxy classification. An object is considered extended if the the difference between the `PSF` magnitude and the [`CModel` magnitude](https://www.sdss.org/dr12/algorithms/magnitudes/#cmodel) is beyond certain threshold (0.0164). To know more about this see [Bosch et al. 2018](http://adsabs.harvard.edu/abs/2018PASJ...70S...5B) section 4.9.10\n",
+    "\n",
+    "**Note** JP writing: I couldn't find the counterpart of `{band}_modelfit_CModel_flux` in dpdd files. So just for the sake of the exercise, I used `magerr_{band}_cModel`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Subset of columns of interest\n",
+    "cols = \"mag_g_cModel, mag_r_cModel, mag_i_cModel\"\n",
+    "\n",
+    "# SQL - register first the DataFrame\n",
+    "df.createOrReplaceTempView(\"full_tract\")\n",
+    "\n",
+    "# Used to be g_modelfit_CModel_flux\n",
+    "sql_command = \"\"\"\n",
+    "    SELECT {}\n",
+    "    FROM full_tract \n",
+    "    WHERE \n",
+    "        tract == 4849 AND\n",
+    "        clean AND\n",
+    "        magerr_g_cModel>0 AND\n",
+    "        magerr_g_cModel<1e16 AND\n",
+    "        magerr_r_cModel>0 AND\n",
+    "        magerr_r_cModel<1e16 AND\n",
+    "        magerr_i_cModel>0 AND\n",
+    "        magerr_i_cModel<1e16\n",
+    "\"\"\".format(cols)\n",
+    "\n",
+    "# Execute the expression - return a DataFrame\n",
+    "df_sub = spark.sql(sql_command)\n",
+    "\n",
+    "print(df_sub.count())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So now, we have selected what we think are stars. Let's take a look at the colors of these objects"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mag_g_cModel, mag_r_cModel, mag_i_cModel = np.transpose(df_sub.collect())\n",
+    "plt.hist2d(mag_g_cModel - mag_r_cModel,\n",
+    "           mag_r_cModel - mag_i_cModel, \n",
+    "           bins=100,range=[(-1,2),(-1,2)]);\n",
+    "plt.xlabel('$g-r$')\n",
+    "plt.ylabel('$r-i$')\n",
+    "plt.colorbar(label='Number of objects');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For more information about the stellar locus from models and in Run1.1p see the [DC2 Coadd Run1.1p direct access -- color-color stellar locus](DC2%20Coadd%20Run1.1p%20direct%20access%20--%20color-color%20stellar%20locus.ipynb) tutorial\n",
+    "\n",
+    "Q: What else can you do to improve the star selection?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__If you want to see more go to [Part II](DC2%20Coadd%20Run1.1p%20GCR%20tutorial%20--%20Part%20II%3A%20Lensing%20Cuts.ipynb)__"
```

Comment:
This part 2 links back to the GCR tutorial -- clarify it here or remove the link.